src/xcode/ENA/ENATests/Helper/TimeInterval+Convenience.swift

```diff
@@ -14,11 +14,13 @@ extension TimeInterval {
 		self = Double(days * 24 * 60 * 60)
 	}
 
-	/// One second
+	/// two seconds
```

Comment:
I like the increase. 👍
Just for discussions sake, why just 2,5 and 10? and not 10, 20, 30? These numbers seem to be arbitrary 🤷‍♂️ If something is time-critical we should consider performance tests and if not, a general timeout of n seconds (for synchronous tasks) would do the trick.